app/Http/Controllers/Game/CreateGameController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers\Game;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Game\CreateGameRequest;
+use App\Services\GameService;
+use Illuminate\Http\JsonResponse;
+
+class CreateGameController extends Controller
+{
+    private $gameService;
+
+    public function __construct()
+    {
+        $this->gameService = new GameService();
```

Comment:
Instead of newing it up in the constructor would you be cool with injecting it `public function __construct(GameService $gameService)`

Reply:
Yup! I would be cool with that!